src/test/kotlin/no/nav/familie/ef/mottak/service/Testdata.kt

```diff
@@ -58,6 +58,50 @@ internal object Testdata {
                                                       arbeidstid = dokumentfelt("Arbeidstid"),
                                                       spesielleBehov = dokumentfelt("Spesielle behov")))
 
+    val søknadSkolepenger = SøknadSkolepenger(Søknadsfelt("Søker", personalia()),
+                                              Søknadsfelt("detaljer", Innsendingsdetaljer(Søknadsfelt("mottat", mottat))),
+                                              Søknadsfelt("Detaljer om sivilstand", sivilstandsdetaljer()),
+                                              Søknadsfelt("Opphold i Norge", medlemskapsdetaljer()),
+                                              Søknadsfelt("Bosituasjonen din", bosituasjon()),
+                                              Søknadsfelt("Sivilstandsplaner", sivilstandsplaner()),
+                                              Søknadsfelt("Arbeid, utdanning og andre aktiviteter", utdanning()),
+                                              dokumentasjon = SkolepengerDokumentasjon(
+
+                                              )
+    )
+
+    private fun utdanning(): UnderUtdanning {
+        return UnderUtdanning(Søknadsfelt("Skole/utdanningssted", "UiO"),
+                              Søknadsfelt("Utdanning",
+                                          Utdanning(Søknadsfelt("Linje/kurs/grad",
+                                                                "Profesjonsstudium Informatikk"),
+                                                    Søknadsfelt("Når skal du være elev/student?",
+                                                                Periode(Month.JANUARY,
+                                                                        1999,
+                                                                        Month.OCTOBER,
+                                                                        2004))
+                                          )),
```

Comment:
Jeg liker ikke linjeskift før parentes slutt. Ikke etter parentes start for den saks skyld.

Reply:
Jeg fikser